js/test/Glacier2/router/Client.ts

```diff
@@ -17,7 +15,7 @@ class CallbackReceiverI extends Test.CallbackReceiver {
         this._p = new Ice.Promise();
     }
 
-    callback(current: Ice.Current) {
+    callback(_: Ice.Current) {
```

Comment:
linter and compiler warn that `current` is unused.

We can:
- use `_` prefix to indicate that is intentional.
- Remove the parameter, even if is part of the paren signature: `callback()`.
- Disable the warnings for unused parameters.